tests/vector/test_sync_vector_env.py

```diff
@@ -167,3 +167,17 @@ def test_custom_space_sync_vector_env():
         "step(action-5)",
         "step(action-7)",
     )
+
+
+def test_sync_vector_env_seed():
+    env = make_env("BipedalWalker-v3", seed=123)()
```

Comment:
Don't hard-code an env, use the parametrization over all envs.

Reply:
Done in b7fc60fdb8a0dc77e8615ffc18dbba668e7b7a14